src/ORM/BehaviorRegistry.php

```diff
@@ -260,12 +260,13 @@ public function call(string $method, array $args = [])
     /**
      * Invoke a finder on a behavior.
      *
+     * @internal
      * @param string $type The finder type to invoke.
-     * @param array $args The arguments you want to invoke the method with.
+     * @param mixed ...$args The arguments you want to invoke the method with.
      * @return \Cake\ORM\Query The return value depends on the underlying behavior method.
      * @throws \BadMethodCallException When the method is unknown.
      */
-    public function callFinder(string $type, array $args = []): Query
+    public function callFinder(string $type, ...$args): Query
```

Comment:
Isn't this also changing the same of `args` as now options are the 0th element instead of the only thing?

Reply:
I'm not really sure why BehaviorRegistry was using an array originally.